java/libphonenumber/src/com/google/i18n/phonenumbers/MetadataManager.java

```diff
@@ -130,10 +104,9 @@ private static void loadAlternateFormatsMetadataFromFile(int countryCallingCode)
       // that they are present, by checking the map of available data first.
       throw new IllegalStateException("missing metadata: " + fileName);
     }
-    PhoneMetadataCollection alternateFormatData =
-        loadMetadataAndCloseInput(source, DEFAULT_BUFFER_SIZE);
-    for (PhoneMetadata metadata : alternateFormatData.metadata) {
-      callingCodeToAlternateFormatsMap.put(metadata.countryCode, metadata);
+    PhoneMetadataCollection alternateFormats = loadMetadataAndCloseInput(source);
```

Comment:
cl/131812609 :)

If you prefer I can change this here later. It's just more consistent with the file name constant and other names in this fie.